src/internal/m365/collection/drive/library_handler.go

```diff
@@ -128,6 +133,7 @@ func (h libraryBackupHandler) NewLocationIDer(
 	driveID string,
 	elems ...string,
 ) details.LocationIDer {
+	// TODO(meain): path related changes for groups
```

Comment:
Can you tell me about how you imagine the groups' path changes might affect the location path here?

Separately, does the TODO make sense as something internal to this func, or should it go into `group_handler` with a todo to override this func?

Reply:
Hmm, I guess this might not require a change according to the current path change design. The change would have been to create a `NewSharePointLocationIDer` for groups and not necessarily this function. I mostly added in the TODO there as I was kinda unsure.